### PR TITLE
Visitor logic 

### PR DESCRIPTION
**Pull request by kermakovich**

### CODE REVIEW

File: src/main/java/solvd/ermakovich/ct/web/controller/VisitorController.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+@RequiredArgsConstructor
+public class VisitorController {
+
+    private final VisitorMapper mapper;
+    private final PerformanceService performanceService;
+    private final VisitorService visitorService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    VisitorDto create(@RequestBody VisitorDto visitorDto) {
```
Comment (h1alexbel) [inline src/main/java/solvd/ermakovich/ct/web/controller/VisitorController.java:35]: @kermakovich why methods are package-private?

Comment (kermakovich) [reply src/main/java/solvd/ermakovich/ct/web/controller/VisitorController.java:35]: It's a mistake, but why should we use public methods? I can say only 1 reason: for tests. Are there any other reasons?

Comment (kermakovich) [reply src/main/java/solvd/ermakovich/ct/web/controller/VisitorController.java:35]: https://stackoverflow.com/questions/38262666/springmvc-controller-public-or-default-protected I found this

Comment (h1alexbel) [reply src/main/java/solvd/ermakovich/ct/web/controller/VisitorController.java]: @kermakovich testing is the most important reason among others

**Pull request merged**